src/pages/Login.js

```diff
@@ -1,9 +1,100 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router';
+import { loginEmail } from '../actions';
+
+export class Login extends React.Component {
```

Comment:
Esse export na linha 7 acaba atrapalhando, pois na linha 96 vc já dá um `export default`. Aí só precisa ajustar a importação no `App.js`. Deixei um comentário por lá também

```suggestion
class Login extends React.Component {
```